src/XenCert/StorageHandler.py

```diff
@@ -1854,6 +1854,13 @@ def Gfs2Supported(self):
                 self.storage_conf['storage_type'] == "fcoe":
             return False
 
+        if pool['restrictions']['restrict_corosync'] == "true":
+            raise Exception("You are required to run these tests on a licensed host, with either XenServer Enterprise "
+                            "or Citrix Virtual Apps or Desktop entitement. This is so that the GFS2 SR can "
```

Comment:
entitement is typo? I could not find the word.

Reply:
It was meant to be "entitlement". Apparently my spelling it terrible when I don't have an automatic spell checker :P

Reply:
No worry, they are extremely small problem